### PR TITLE
rfcs/23/README.md: fix typos

### PR DESCRIPTION
**Pull request by nodakai**

### CODE REVIEW

File: content/docs/rfcs/23/README.md [hunk 1]
```diff
@@ -355,11 +355,6 @@ The implementation SHOULD follow http://rfc.zeromq.org/spec:29/PUBSUB for the se
 
 When using ZMTP, message filtering SHALL happen at the publisher side (the PUB or XPUB socket). To create a subscription, the SUB or XSUB peer SHALL send a SUBSCRIBE message, which has this grammar:
 
-message = *message-more message-last
```
Comment (bluca) [inline content/docs/rfcs/23/README.md:358]: why is this block being removed?

Comment (kai-alpha) [reply content/docs/rfcs/23/README.md:358]: It has nothing to do with subscription per se, right?

I don't know why that block was copied from 

https://github.com/zeromq/rfc/blob/f8160affb154edea301d2fd1a912d4289e921c1a/content/docs/rfcs/23/README.md#L124

without `<code>` formatting

Comment (nodakai) [reply content/docs/rfcs/23/README.md:358]: It has nothing to do with subscription per se, right?

I don't know why the block was copied from

https://github.com/zeromq/rfc/blob/f8160affb154edea301d2fd1a912d4289e921c1a/content/docs/rfcs/23/README.md#L124

without `<code>` formatting

Comment (bluca) [reply content/docs/rfcs/23/README.md:358]: Ah, I see what you mean - looked at the diff only, not the rendered markdown

**Pull request merged**